index.js

```diff
@@ -304,7 +304,10 @@ exports.logger = function logger(options) {
                 chalk.grey("{{res.responseTime}}ms");
               coloredRes.statusCode = chalk[statusColor](res.statusCode);
             }
-            var msgFormat = !options.expressFormat ? options.msg : expressMsgFormat;
+
+            var msgFormat = !options.expressFormat ?
+              typeof options.msg === 'function' ? options.msg(req, res) : options.msg
+              : expressMsgFormat
```

Comment:
can you make it there is not a double ternary?

Reply:
you want it with if/elses?